tests/go.mod

```diff
@@ -0,0 +1,106 @@
+module github.com/terraform-ibm-modules/terraform-ibm-hpcs
+
+go 1.20
+
+require (
+	github.com/stretchr/testify v1.8.2
+	github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper v1.8.11
```

Comment:
Can this be upgraded to a latest version?  We noticed a vulnerability and we had to update.  So please update here too.  (Though I am not sure why it was not detected in the pipeline.)

Reply:
I think that pipeline does not check the module versions so not caught. (not sure though) 
renovate checks in main branch and updates. Made changes now. Thanks